pages/sdk/relay-kit.mdx

```diff
@@ -3,7 +3,7 @@ import CustomCard from '../../components/CustomCard'
 
 # Relay Kit
 
-The [Relay Kit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/relay-kit) enables ERC-4337 with Safe, and allows users to pay for the transaction fees from their Safe account using the blockchain native token, ERC-20 tokens, or to get their transactions sponsored.
+The Relay Kit enables ERC-4337 with Safe, and allows users to pay for the transaction fees from their Safe account using the blockchain native token, ERC-20 tokens, or to get their transactions sponsored.
```

Comment:
Can we also incorporate [this feedback](https://5afe.slack.com/archives/C0608CC95JT/p1723645305652529?thread_ts=1723644579.599529&cid=C0608CC95JT)?

Reply:
This would be out of scope for this PR